json-rpc/src/runtime.rs

```diff
@@ -26,6 +28,33 @@ const LABEL_SUCCESS: &str = "success";
 const LABEL_BATCH: &str = "batch";
 const LABEL_SINGLE: &str = "single";
 
+#[derive(Schema)]
+struct HttpRequestLog<'a> {
+    #[schema(display)]
+    remote_addr: Option<std::net::SocketAddr>,
+    method: String,
+    path: String,
+    status: u16,
+    referer: Option<&'a str>,
+    user_agent: Option<&'a str>,
+    #[schema(debug)]
+    elapsed: std::time::Duration,
+}
+
+#[derive(Schema)]
+struct RpcRequestLog {
+    trace_id: u64,
```

Comment:
I'd call this a `request_id`.

Reply:
request_id maybe confusing because json-rpc protocol has request { id ... }, hence give trace_id.
also, for batch request, we will only have one trace id for all json-rpc requests

Reply:
Ah interesting okay.  So, not telling any difference between any requests on a batch command